nvflare/app_common/abstract/model_persistor.py

```diff
@@ -20,6 +20,20 @@
 from .learnable_persistor import LearnablePersistor
 
 
+class ModelDescriptor:
```

Comment:
This class ModelDescriptor should be moved to a separate file (say model_desc.py) under app_common. ModelDescriptor could be used for other purposes too.